src/chart/bar/BarView.js

```diff
@@ -205,6 +229,15 @@ export default echarts.extendChartView({
             })
             .execute();
 
+        var bgGroup = group.childOfName('background') || new Group({ name: 'background' });
```

Comment:
It might not be a efficient way if using `childOfName` since all of the bar `el` are added to `group`.
I think it is OK to mount it to `this`.